### ISSUE TITLE
can't get work with rn 0.31.0

### ISSUE THREAD

**Issue by cinder92** [tlenclos/react-native-audio-streaming#25]
Hello, thanks for your awesome plugin but i can not get work with rn 0.31.0, this is my package json

```
"dependencies": {
    "moment": "^2.13.0",
    "react": "15.2.1",
    "react-native": "0.31.0",
    "react-native-maps": "^0.7.1",
    "react-native-scrollable-tab-view": "^0.5.4",
    "react-native-share": "^1.0.16",
    "react-native-swiper": "^1.4.11",
    "react-native-vector-icons": "^2.0.3",
    "react-redux": "^4.4.5",
    "redux": "^3.5.2",
    "redux-thunk": "^2.1.0"
  }
```

logs

```
iMac-de-Dante:MiDiocesis dantecervantes$ npm install react-native-audio-streaming --save
npm WARN peerDependencies The peer dependency react-native@^0.30.0 included from react-native-audio-streaming will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 16.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "react-native-audio-streaming" "--save"
npm ERR! node v4.4.4
npm ERR! npm  v2.15.1
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react-native@0.31.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-native-scrollable-tab-view@0.5.4 wants react-native@>=0.20.0
npm ERR! peerinvalid Peer react-native-audio-streaming@2.1.0 wants react-native@^0.30.0

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dantecervantes/Documents/react-native/MiDiocesis/npm-debug.log

```

Thanks!

**Comment by davidsharp** [comment]
+1

Having the same problem with React Native 0.33.0

**Comment by hounsellal** [comment]
Same problem with RN 0.35.0

**Comment by tlenclos** [comment]
This should be fixed with latest release (2.2.0), see https://github.com/tlenclos/react-native-audio-streaming/commit/79ef1dbe1031a198c183a480b2441c853da4ac14.
